### PR TITLE
netserver: implement RTM_GETROUTE rtnetlink request

### PR DESCRIPTION
**Pull request by Geertiebear**
~~Contains some weird stuff~~ Should be fixed.

### CODE REVIEW

**Comment by Dennisbonke** [comment]
Bump on this, can we get this merged in?

**Review by no92 — APPROVED** [review]
Other than the minor issues pointed out, LGTM.

File: posix/subsystem/src/sockutil.hpp [hunk 1]
```diff
@@ -1,44 +0,0 @@
-#pragma once
```
Comment (no92) [inline posix/subsystem/src/sockutil.hpp:1]: There are still multiple places including this file.

**Review by no92 — CHANGES_REQUESTED** [review]

**Comment by no92** [comment]
Closing as #510 is merged now.

**Pull request closed without merging**